grid2op/multi_agent/multiAgentEnv.py

```diff
@@ -471,6 +528,9 @@ def _build_subgrid_cls_from_domain(self, domain):
         res_cls.assert_grid_correct_cls()
         return res_cls
     
+    #TODO BEN: check that cent env is initialized and not closed
+    #TODO BEN: update self.observations
+        
```

Comment:
The location of these comments does not make any sense, clearly... But that's your problem not mine. Hopefully you won't run into hard to find bugs in a few days when i'll be away because of that...

Reply:
It is done in future versions